app/src/ui/changes/changes-list.tsx

```diff
@@ -31,6 +31,11 @@ interface IChangesListProps {
    * @param path The path of the file relative to the root of the repository
    */
   readonly onRevealInFileManager: (path: string) => void
+  /**
```

Comment:
At the start of each comment?

`  readonly onDiscardChanges: (path: string) => void

  /**
   * Called to reveal a file in the native file manager.
   * @param path The path of the file relative to the root of the repository
   */

  readonly onRevealInFileManager: (path: string) => void
  /**
   * Called to open a file it its default application
   * @param path The path of the file relative to the root of the repository
   */
  readonly onOpenItem: (path: string) => void`